unity/EditorPlugin/PluginEntryPoint.cs

```diff
@@ -333,6 +334,15 @@ private static void CreateProtocolAndAdvise(Lifetime lifetime, List<ProtocolInst
       }
     }
 
+    private static void GetBuildLocation(EditorPluginModel model)
+    {
+        var path = EditorUserBuildSettings.GetBuildLocation(EditorUserBuildSettings.selectedStandaloneTarget);
```

Comment:
What happens if a non-standalone player is selected, such as iPhone or Android?

Reply:
everything except non-standalone should be ignored, I haven't checked that though.

Reply:
just checked. when android is selected - nothing happens, our standalone configuration is not generated, which is ok.